lib/manager/components/DeploymentViewer.js

```diff
@@ -102,6 +111,33 @@ export default class DeploymentViewer extends Component<Props, State> {
     return [[north, east], [south, west]]
   }
 
+  _loadOptions = () => {
+    fetch('https://r5-builds.s3.amazonaws.com')
```

Comment:
This should probably be a config value instead of being hardcoded.

Reply:
Do you think the bucket name or the entire URL should be configurable? I'd lean towards bucket name.